src/Test.UnitTests.BinaryParsers/PEBinary/PEBinaryTests.cs

```diff
@@ -32,27 +32,40 @@ internal static string GetTestDirectory(string relativeDirectory)
             return Path.Combine(dirPath, relativeDirectory);
         }
 
-        [Fact]
-        public void PEBinary_PdbAvailable()
+        [Theory]
+        [InlineData("clangcl.14.pe.c.codeview.pdbpagesize_default.exe")]
+        [InlineData("clangcl.14.pe.c.codeview.pdbpagesize_4096.exe")]
+        [InlineData("clangcl.14.pe.c.codeview.pdbpagesize_8192.exe")]
+        [InlineData("clangcl.14.pe.c.codeview.pdbpagesize_16384.exe")]
```

Comment:
>16384 [](http://example.com/codeflow?start=58&length=5)

these are not random test, it should be all the supported number:
/PdbPageSize:[4096|8192|16384|32768]
the difference between default and 4096 is whether it is explicitly passed.